apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -166,6 +171,7 @@ start_ae(StakersEncoded, PCSpendAddress) ->
             end,
             StakersEncoded),
     StakersMap = maps:from_list(Stakers),
+    lager:warning("Stakers ~p", [StakersMap]),
```

Comment:
Is this really a warning? I would say it is more an info level log

Reply:
Sorry that was for my debugging. I'll remove it completely.